.env

```diff
@@ -1,2 +1,4 @@
 INCLUDE_MAKEFILE=terraform.inc
-#REMOTE_MAKEFILE=https://raw.githubusercontent.com/<githubuser>/<repository>/<branch_or_tag>/${INCLUDE_MAKEFILE}
\ No newline at end of file
+#REMOTE_MAKEFILE=https://raw.githubusercontent.com/<githubuser>/<repository>/<branch_or_tag>/${INCLUDE_MAKEFILE}
```

Comment:
Existe algum motivo para não usarmos um valor default aqui sem ta comentado?

Reply:
Proposta de alteração:

INCLUDE_MAKEFILE=how-to-use.inc
> Deixar como padrão um arquivo `how-to-use.inc`, explicando (através da execução do comando `make` ou `make help`) como usar o Makefile base e sugerindo a mudança para outras tecnologias.

VERSION_MAKEFILE=main
> Incluir um variável para controlar a versão do projeto templates. O valor default será `main`, mas no README será recomendado não usá-lo, e sim usar tags.

REMOTE_MAKEFILE=https://raw.githubusercontent.com/mentoriaiac/Makefiles/${VERSION_MAKEFILE}/${INCLUDE_MAKEFILE}
> Deixar o caminho do repo corrente como default.

Reply:
Tem um [PR](https://github.com/marcelomansur/Makefiles/pull/1) dentro do meu fork do projeto, com a implementação destas propostas. Caso estejam de acordo com as propostas, atualizo este daqui.

Reply:
PR atualizado.